### PR TITLE
Add tests for CompletedTasksTable

### PR DESCRIPTION
**Pull request by johnsonm325**

### CODE REVIEW

File: src/PresentationalComponents/ExecuteTaskButton/ExecuteTaskButton.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ ExecuteTaskButton.propTypes = {
   setIsRunTaskAgain: propTypes.func,
   setModalOpened: propTypes.func,
   slug: propTypes.string,
-  title: propTypes.string,
+  title: propTypes.any,
```
Comment (mkholjuraev) [inline src/PresentationalComponents/ExecuteTaskButton/ExecuteTaskButton.js:64]: I think that the title might be either string or a React element. If this is the case, can we use more explicit prop type. 

```suggestion
  title: PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.elementType,
  ])
```

File: src/SmartComponents/RunTaskModal/RunTaskModal.js [hunk 2]
```diff
@@ -155,7 +155,7 @@ RunTaskModal.propTypes = {
   setIsRunTaskAgain: propTypes.func,
   setModalOpened: propTypes.func,
   slug: propTypes.string,
-  title: propTypes.string,
+  title: propTypes.any,
```
Comment (mkholjuraev) [inline src/SmartComponents/RunTaskModal/RunTaskModal.js:158]: Here also, a more explicit prop type may be used.

**Review by mkholjuraev — APPROVED** [review]
LGTM!

**Review by Fewwy — APPROVED** [review]
@johnsonm325 LGTM!

**Pull request merged**